rest-api-spec/src/main/resources/rest-api-spec/api/ml.start_trained_model_deployment.json

```diff
@@ -33,6 +33,12 @@
         "required":false,
         "description":"Controls the amount of time to wait for the model to deploy.",
         "default": "20s"
+      },
+      "wait_for":{
+        "type":"string",
+        "required":false,
+        "description":"The allocation state for which to wait. Valid values are: starting, partially_started, started",
```

Comment:
Since this is an enum can we add the `options` field with all the valid choices? ([example of this](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/indices.get.json#L41))

Reply:
boom, TIL about the `options` field :D

Reply:
:muscle: